File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        string sortedS = s, sortedT = t;
        sort(sortedS.begin(), sortedS.end());
        sort(sortedT.begin(), sortedT.end());

        if(sortedS == sortedT){
            return true;
        }else{
            return false;
        }
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        map = {}

        for i in nums:
            map[i] = 1 + map.get(i, 0)

        arr = []
        for i, c in map.items():
            arr.append([c, i])

        arr.sort()
        
        res = []
        while len(res) < k:
            res.append(arr.pop()[1])
        return res